### PR TITLE
Add GSet implementation

### PR DESCRIPTION
**Pull request by vitorenesduarte**

### CODE REVIEW

File: src/gset.erl [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+-spec delta_mutate(gset_op(), type:actor(), gset()) ->
+    {ok, delta_gset()}.
+delta_mutate({add, Elem}, _Actor, {?TYPE, GSet}) ->
+    Delta = case ordsets:is_element(Elem, GSet) of
+        true ->
+            ordsets:new();
+        false ->
+            ordsets:add_element(Elem, ordsets:new())
+    end,
+    {ok, {delta, {?TYPE, Delta}}}.
```
Comment (cmeiklejohn) [inline src/gset.erl]: This doesn't seem to match the type.

Comment (vitorenesduarte) [inline src/gset.erl]: I was looking to GCounter impl when doing this one.
It's fixed.

**Comment by cmeiklejohn** [comment]
Are you still working on this or is it good to go?

**Comment by cmeiklejohn** [comment]
Closed without merge?

**Pull request closed without merging**